lib/compile/csdl2openapi.js

```diff
@@ -2249,9 +2249,6 @@ see [Expand](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-prot
                 s.pattern = "^'([^']|'')*'$";
             }
             if (element.$Nullable) {
-                if (!s.anyOf && !s.allOf) s.type = "string";
```

Comment:
Removing this would change the logic for allOf and anyOf not specified too?
isn't it like if format is specified we should return format instead of adding null object?